R/FilterStateRange.R

```diff
@@ -271,13 +280,8 @@ RangeFilterState <- R6::R6Class( # nolint
       check_in_range(value, private$choices, pre_msg = pre_msg)
     },
     cast_and_validate = function(values) {
-      tryCatch(
-        expr = {
-          values <- as.numeric(values)
-          if (any(is.na(values))) stop()
-        },
-        error = function(error) stop("The array of set values must contain values coercible to numeric.")
-      )
+      values <- as.numeric(values)
```

Comment:
do we need a suppresswarnings here?

Reply:
In the previous version these warnings were never actually triggered.
They could be suppressed or they could be logged, depending on how we want to report the set range.